src/components/DailyStatistics/TitleStatistic.tsx

```diff
@@ -61,15 +62,19 @@ export const TitleStatisticComponent: FunctionComponent<TitleStatisticComponent>
   onPressPrevDay,
   onPressNextDay,
 }) => {
+  const { i18nt } = useTranslate();
   return (
     <View style={styles.appHeaderWrapper}>
-      <AppText style={styles.smallText}>You distributed</AppText>
+      <AppText style={styles.smallText}>
+        {i18nt("redemptionStats", "youDistributed")}
+      </AppText>
       <AppText style={styles.statText}>{totalCount?.toLocaleString()}</AppText>
       <AppText style={styles.smallText}>
-        Last distributed at{" "}
+        {i18nt("redemptionStats", "lastDistributedAt")}{" "}
         {lastTransactionTime !== null
           ? format(lastTransactionTime, "h:mma")
-          : "-"}
+          : "-"}{" "}
+        {i18nt("redemptionStats", "distributedTime")}
```

Comment:
This `i18nt` copy can be moved to line 75, since it is possible to feed dynamic values into `i18nt` for interpolation. You can refer to existing implementations of `i18nt` for that.

This can be used to show different copies for `en` and `zh`, where en only has the `lastTransactionTime`, but `zh` has the `lastTransactionTime` and additional text.